src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx

```diff
@@ -227,6 +235,13 @@ public:
      */
     void sock_event_handler(void *context);
 
+    /** This function handles http server callback indication.  This is public
+     * only so that an extern "C" method can call it.  DO NOT use directly.
+     * @param context1 pointer to HTTP Server Event info
+     * @param context2 pointer to HTTP Server Response info
+     */
+    void http_server_callback(void *context1, void *context2);
```

Comment:
these should not be void*
If you're worried about the dependency intriduced by the inclusion of simplelink headers, consider forward declaring the proper type in this file and set these pointers to be properly typed.
Please change the other callbacks similarly.